example/src/index.css

```diff
@@ -41,4 +41,14 @@ footer {
 		overflow: hidden;
     clip-path: polygon(0 0, 100% 0%, 115% 85%, 0 100%);
   }
+}
+
+.section {
```

Comment:
Why don't we use Tailwind for this?

Reply:
I think we still don't use Tailwind? I don';t think we need any custom css classes? Thats not the Tailwind way to do things?